firefox_puppeteer/api/appinfo.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class AppInfo(BaseLib):
+
+    @property
+    def browserTabsRemoteAutostart(self):
```

Comment:
Would you mind to file a bug for getattr, which describes exactly what to do? It would be a good bug for a contributor starting on our project. Thanks.